src/element-service.js

```diff
@@ -219,7 +219,7 @@ function waitForExtensionIfPresent(win, extension, head) {
    * not present, or the service was defined by a test. In those cases
    * we don't wait around for an extension that does not exist.
    */
-  if (isElementScheduled(win, extension) ||
+  if (!isElementScheduled(win, extension) ||
```

Comment:
`&&`

Reply:
doh!

Reply:
`isElementScheduled` is "scheduled" if the element is stubbed, so we can't guarantee that the extension script has run yet. Moving the `extensionScriptsInNode` call into this function was specifically so we can avoid it when we know it's scheduled.